docs/src/app/components/pages/components/Table/ExampleComplex.jsx

```diff
@@ -117,7 +123,10 @@ export default class TableExampleComplex extends React.Component {
               </TableRow>
               ))}
           </TableBody>
-          <TableFooter>
+          <TableFooter
+            displaySelectAll={this.state.showCheckboxes}
```

Comment:
We only have the documentation for the `TableHeader`. What's wrong here, this line or the doc :grin:?